src/__test__/reduce/missionslice.test.js

```diff
@@ -0,0 +1,53 @@
+import axios from 'axios';
+import store from '../../redux/store';
+import { fetchMission, joinMission, leaveMission } from '../../redux/mission/missionSlice';
+
+describe('Mission redux state tests', () => {
+  it('Should initially set Mission store to an empty Array', () => {
+    const state = store.getState().mission.missions;
+    expect(state).toEqual([]);
+  });
+
+  it('should Join Mission payload send correct', () => {
+    const expectedPayload = { payload: '9D1B7E0', type: 'mission/joinMission' };
+    const actualPayload = joinMission('9D1B7E0');
+    expect(actualPayload).toEqual(expectedPayload);
+  });
+
+  it('should Leave Mission payload send correct', () => {
+    const expectedPayload = { payload: '9D1B7E0', type: 'mission/leaveMission' };
+    const actualPayload = leaveMission('9D1B7E0');
+    expect(actualPayload).toEqual(expectedPayload);
+  });
+
+  it('fetches data from API', async () => {
+    const url = 'https://api.spacexdata.com/v3/missions';
+    const axiosSpy = jest.spyOn(axios, 'get');
+    jest.setTimeout(90000);
+    const dispatchSpy = jest.fn();
+    await fetchMission(url)(dispatchSpy);
+    expect(axiosSpy).toHaveBeenCalledWith(url);
+  });
+});
+
+//
+//   describe('Unit testing for calculate.js', () => {
+//     test('First time pressing +', () => {
+//       const myMockedObject = {
+//         total: null,
+//         next: null,
+//         operation: null,
+//       };
+
+//       let MockedButtonName = null;
+
+//       myMockedObject.total = '100';
+//       MockedButtonName = '+';
+
+//       expect(calculate(myMockedObject, MockedButtonName)).toStrictEqual({
+//         total: '100',
+//         next: null,
+//         operation: '+',
+//       });
+//     });
+// });
```

Comment:
- [ ] while ur project is perfect and all of your tests are green, unfortunately i can't approve the project while it have some commented code , kindly remove it to keep your code professional and clean 🙏